### PR TITLE
feat(checkout): Add purchuse legend when the buy is by credits

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Show new purchuse legends when the buy is by credits.

**Screenshots:**

**Before:**

**Credit Card:**

image

**Transfer:**

image

**After:**

**Credit Card:**

image

**Transfer:**

image

The refenrecen abot the changes in the legends are in: [Nuevo Flujo de Checkout](https://makingsense.atlassian.net/wiki/spaces/DOP/pages/1453785101/Nuevo+Flujo+de+Checkout)

**Task:** [DAT-676](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-676)

### CODE REVIEW

**Review by silsanchez8 — COMMENTED** [review]
Good work! 💪  I left just a comment.

File: src/components/ChangePlanDeprecated/Checkout/PurchaseSummary/PurchaseSummary.js [hunk 1]
```diff
@@ -170,10 +170,26 @@ export const Promocode = () => {
...
 };
 
-export const InvoiceInformation = ({ priceToPay, discount, paymentMethodType }) => {
+export const InvoiceInformation = ({ priceToPay, discount, paymentMethodType, planType }) => {
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
 
+  const getTaxesLegend = (paymentMethodType, planType) => {
+    switch (planType) {
+      case 'prepaid':
```
[the remark this answers is not in the document]
Comment (jhoffman-ms) [reply src/components/ChangePlanDeprecated/Checkout/PurchaseSummary/PurchaseSummary.js:179]: Fine, I will use it.

[the remark this answers is not in the document]
Comment (jhoffman-ms) [reply src/components/ChangePlanDeprecated/Checkout/PurchaseSummary/PurchaseSummary.js]: Done. Thanks @silsanchez8

**Pull request merged**